ts/Data/Formula/FormulaParser.ts

```diff
@@ -572,7 +572,18 @@ function parseFormula(
         // Check for a number value
         match = next.match(decimalRegExp);
         if (match) {
-            formula.push(parseFloat(match[0]));
+            let number = parseFloat(match[0]);
+            // If the current value is multiplication-related and the previous
+            // one is a minus sign, set the current value to negative and remove
+            // the minus sign.
+            if (
+                formula[formula.length - 1] === '-' &&
+                formula[formula.length - 2] === '*'
```

Comment:
This is not universal enough. Check other operators than only `*`, e.g. `/` or `^`. Do we even need to check what specifically it is? Maybe `length > 1` and `last == '-'` is enough?

Reply:
The sequence for the negative value multiplication is basically `['*', '-', number]`. I don't like those specific checks either, but the issue is that if we check for the `'-'` only, we exclude the simple subtraction, so that's why we also check for the `'*'`. To sum up, the issue is to distinguish subtraction from multiplication by a negative value at some level (I'm saying that because I was thinking about handling it at the regex/extracting level, where we build the proper formula). 

Additionally, we can have multiple multiplications, such as `-x * -y * -z`, so I think it's the safest option to always base it on the previous elements from the particular iteration.

Reply:
As for the multiplication,`'B1 * -C1'` also doesn't work, but it should.
![image](https://github.com/user-attachments/assets/d1809660-2ff4-4797-af24-cb766cbf3600)

We are currently patching too small part of the bigger problem in my opinion.